internetarchive/iaux-icons
# Add new packages for sorting icons

### Changes:

- add sort descending and sort ascending icons


### To Test:

- pull down repo -> go to directory: packages/ia-icons
- in packages/ia-icons directory, run command in-terminal: npm install && npm start
- local server pops up with icon list (random colors generated)
- you should see sortAscending and sortDescending icon in the list when you hover to it 

### On first load look:
<img width="223" alt="Screen Shot 2021-05-22 at 1 40 12 AM" src="https://user-images.githubusercontent.com/1281581/119177441-e84ea780-ba9e-11eb-90ac-746d44b8c241.png">

### On mouse-hover look:
<img width="99" alt="Screen Shot 2021-05-22 at 1 42 26 AM" src="https://user-images.githubusercontent.com/1281581/119177505-fdc3d180-ba9e-11eb-9ad9-24f22c21217a.png">
<img width="102" alt="Screen Shot 2021-05-22 at 1 42 19 AM" src="https://user-images.githubusercontent.com/1281581/119177510-fef4fe80-ba9e-11eb-8c67-e570802bf4d3.png">
